Insights/Host/HostIPStats.yaml

```diff
@@ -0,0 +1,349 @@
+  | where isnotempty(DestIP_s) and isnotempty(SrcPublicIPs_s)
+  | extend SrcPublicIPs_s = split(SrcPublicIPs_s, ' ')
+  | mv-expand SrcPublicIPs_s
+  | extend SrcPublicIPs_s = tostring(split(SrcPublicIPs_s, '|')[0])
+  | where DestIP_s != SrcPublicIPs_s
+  | extend Direction = "inbound"
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), InboundCount = count() by HostIPAddress = DestIP_s, Type, RemoteIPAddress = SrcPublicIPs_s, Direction, ReceivedBytes = tolong(InboundBytes_d)
+  ),
+  (
+  CommonSecurityLog
+  | where TimeGenerated between (start_time..end_time)
+  | where SourceIP in (IpList)
+  | where SourceIP != DestinationIP
+  //| where DeviceAction has_any ('allow', 'allowed', 'accept', 'built', 'start', 'connect', '')
+  //| where not(DeviceAction has_any ('built','deny', 'denied', 'rst', 'blocked', 'teardown'))
+  | extend Direction = iff(CommunicationDirection !in~ ('outbound','0') or CommunicationDirection !in~ ('inbound','1'), 'NotAvailable', CommunicationDirection)
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), InboundCount = countif(Direction in~ ('Inbound','1')), OutboundCount = countif(Direction in~ ('Outbound', '0')), UnknownDirection = countif(Direction =~ 'NotAvailable') by HostIPAddress = SourceIP, Type = strcat(Type,':', DeviceVendor,'-', DeviceProduct), RemoteIPAddress = DestinationIP, Direction, SentBytes = tolong(SentBytes)
+  ),
+  (
+  CommonSecurityLog
+  | where TimeGenerated between (start_time..end_time)
+  | where DestinationIP in (IpList)
+  | where SourceIP != DestinationIP
+  //| where DeviceAction has_any ('allow', 'allowed', 'accept', 'built', 'start', 'connect', '')
+  //| where not(DeviceAction has_any ('built','deny', 'denied', 'rst', 'blocked', 'teardown'))
+  | extend Direction = iff(CommunicationDirection !in~ ('outbound','0') or CommunicationDirection !in~ ('inbound','1'), 'NotAvailable', CommunicationDirection)
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), InboundCount = countif(Direction in~ ('Inbound','1')), OutboundCount = countif(Direction in~ ('Outbound', '0')), UnknownDirection = countif(Direction =~ 'NotAvailable') by HostIPAddress = DestinationIP, Type = strcat(Type,':', DeviceVendor,'-', DeviceProduct), RemoteIPAddress = SourceIP, Direction, ReceivedBytes = tolong(ReceivedBytes)
+  )
+  );
+  IpStats | join kind=leftouter Host_Account on HostIPAddress | project-away HostIPAddress1
+  };
+  GetIPResolutionWithHost('{{Host_HostName}}', '{{Host_NTDomain}}', '{{Host_DnsDomain}}', '{{Host_AzureID}}', '{{Host_OMSAgentID}}')
+
+Insights:
+ Id: 2ed19a9a-7ebe-4941-a6a0-fa0b020c48da
+ DisplayName: "Host IP address remote connections"
+ Description: |
+  'Provides the remote IP connection count information for outbound and inbound connections to an IP address associated with a given host. Note due to potential performance impact, data is limited to a 7 day max window.'
+ DefaultTimeRange: 
```

Comment:
The disclaimer states that data is "7 days" max, but time range is 12h. Since currently Insights are not configurable - is the disclaimer needed (or do we want to change the time range?)

Reply:
The slider on the entity page will change the query timespan, so while for Investigation Graph it will only use 12h, for Entity Page we need to control the max range.